### PR TITLE
docs: Add missing document link

### LINKED ISSUE #557: Add missing document link
The cpuset doc is unreferenced and hence not very visible to users.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add link in the README to the existing cpuset doc which previously was not very visible.

Fixes #557.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
I think we can avoid a CI run for this one :smile:

**Comment by bergwolf** [comment]
> I think we can avoid a CI run for this one 😄

We don't have the super power to force merge now :)

**Comment by grahamwhaley** [comment]
hmm, metrics hit a build time fail - let me re-nudge:
```
10:49:40 Install CRI-O dependencies available for Ubuntu 16.04
10:49:42 FATAL: command execution failed
```

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * [Developer mode](#developer-mode)
 * [Enable trace support](#enable-trace-support)
 * [Enable debug console](#enable-debug-console)
+* [cpuset cgroup details](#cpuset-cgroup-details)
```
Comment (grahamwhaley) [inline README.md]: nit, as you like nits....
`cpuset` is not capitalised, and the rest of the entries are - which may be as it is a 'proper command', in which case maybe make it a `code block style`.

Comment (jodh-intel) [reply README.md]: Yep - code block style looks best to me. Updated.

**Comment by jodh-intel** [comment]
Ugh - 5 line doc change requiring full CI run...

/test

**Comment by jodh-intel** [comment]
Nobody looking? Good...

**Comment by devimc** [comment]
@jodh-intel :smile: 

form https://github.com/kata-containers/tests/pull/1583#issuecomment-493076736

> @devimc - did you just merge your own PR? :-)
Not something we normally do on the kata repos...
I think that's the second (not just you...) I've seen in the last 24h - let's keep an eye on that practice folks....

cc @grahamwhaley

**Comment by grahamwhaley** [comment]
yeah, I know @devimc , but I am just one voice.... if others in the project feel they need to merge their own PRs to aid project velocity, then, well, we should probably agree at a project level and document that that is acceptable. Historically for the last 3+ years on this and some of its forerunner codebases though, it was not the norm. I'll likely still refrain from merging my own though :-)

**Comment by jodh-intel** [comment]
@grahamwhaley - I don't believe we've actually documented either way? If all checks are passed, I don't personally see the problem (why do we need to wait for someone else to notice it's ready to merge when it's "ready to merge" and the author is happy to "do that work"? :)

However, if it offends sensibilities, do feel free to raise at the Arch Committee meeting so we can thrash it out.

**Pull request merged**